### PR TITLE
Update Payment API

### PR DESCRIPTION
**Pull request by sembrestels**
I included those missing commands to the API reference: `z_exportviewingkey`, `z_importviewingkey`, and `z_listunspent`.

Do you think we should include `z_mergetoaddress`, `z_getpaymentdisclosure`, and `z_validatepaymentdisclosure` too, despite they are experimental?

### CODE REVIEW

File: doc/payment-api.md [hunk 1]
```diff
@@ -65,12 +65,16 @@ z_exportkey | zaddr | _Requires an unlocked wallet or an unencrypted wallet._<br
...
 z_importwallet | filename | _Requires an unlocked wallet or an unencrypted wallet._<br><br>Imports private keys from a file in wallet export file format (see z_exportwallet). These keys will be added to the keys currently in the wallet. This call may need to rescan all or parts of the block chain for transactions affecting the newly-added keys, which may take several minutes.<br><br>Filename is the file to import. The path is relative to zcashd’s working directory.<br><br>No value is returned but a JSON-RPC error will be reported if a failure occurred.
+z_exportviewingkey | zaddr | Reveals the viewing key corresponding to 'zaddr'. Then the z_importviewingkey can be used with this output.
+z_importviewingkey | vkey [rescan=whenkeyisnew] [startHeight=0] | Adds a viewing key (as returned by z_exportviewingkey) to your wallet.
```
Comment (mdr0id) [inline doc/payment-api.md]: Add clarification to startHeight param

**Comment by mdr0id** [comment]
ACK

**Comment by mdr0id** [comment]
@zkbot r+

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**